source/manual/incident-fixing.html.md

```diff
@@ -0,0 +1,21 @@
+---
+owner_slack: "@ana.botto"
+title: Fix an Incident
+parent: "/manual.html"
+layout: manual_layout
+type: learn
+section: Incidents
+---
+
+When there is an incident in an application you are supporting, it is important to make a quick fix so that users can get back to use the application as usual.
```

Comment:
There is existing documentation on responding to an incident - the two should be linked to each other for easy access and discovery in a crisis